packages/apps/explorer/package.json

```diff
@@ -1,18 +1,20 @@
 {
-  "name": "@kadena/block-explorer",
+  "name": "@kadena/explorer",
   "version": "1.0.0",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "test": "echo test",
+    "test": "",
+    "lint-staged": "lint-staged",
     "prisma:generate": "npx prisma generate",
     "create:info:test": "curl --request GET 'http://web:3000/api/info/create?network=Testnet'",
     "create:info:main": "curl --request GET 'http://web:3000/api/info/create?network=Mainnet'"
   },
```

Comment:
Could be that the fixed ci requires two new commands `_phase:build` and `_phase:test`. But as this is an app, and not a lib or tool I like it better if there's no phases here.

Reply:
I've left these out for now